Distribution/Data/skills.json

```diff
@@ -3,566 +3,622 @@
     },
     "Magery": {
-      "Delay": 0,
-      "IntAdvancement": {
-        "Chance": 100,
-        "PointsAmount": 5,
-        "PointsSides": 8,
-        "PointsBonus": 18
-      },
-      "DefaultPoints": 300
+      "Delay": "00:00:00",
+      "DefaultPoints": 300,
+      "StatAdvancements": {
+        "Int": {
+          "Chance": 1,
+          "MinGain": 13,
+          "MaxGain": 58
+        }
+      }
     },
     "MagicResist": {
-      "Delay": 0,
-      "IntAdvancement": {
-        "Chance": 50,
-        "PointsAmount": 3,
-        "PointsSides": 4,
-        "PointsBonus": 15
-      },
-      "DefaultPoints": 50
+      "Delay": "00:00:00",
+      "DefaultPoints": 50,
+      "StatAdvancements": {
+        "Int": {
+          "Chance": 0.5,
+          "MinGain": 7,
+          "MaxGain": 27
+        }
+      }
     },
     "Meditation": {
-      "Delay": 10,
-      "IntAdvancement": {
-        "Chance": 70,
-        "PointsAmount": 5,
-        "PointsSides": 5,
-        "PointsBonus": 15
-      },
-      "DefaultPoints": 250
+      "OnUseHandler": "Server.SkillHandlers.Meditation",
+      "Delay": "00:00:10",
+      "DefaultPoints": 250,
+      "StatAdvancements": {
+        "Int": {
+          "Chance": 0.7,
+          "MinGain": 10,
+          "MaxGain": 40
+        }
+      }
     },
     "SpiritSpeak": {
-      "Delay": 10,
-      "IntAdvancement": {
-        "Chance": 100,
-        "PointsAmount": 5,
-        "PointsSides": 8,
-        "PointsBonus": 35
-      },
-      "DefaultPoints": 20
+      "OnUseHandler": "Server.SkillHandlers.SpiritSpeak",
+      "Delay": "00:00:10",
+      "DefaultPoints": 20,
+      "StatAdvancements": {
+        "Int": {
+          "Chance": 1,
+          "MinGain": 13,
+          "MaxGain": 75
+        }
+      }
     },
     "AnimalLore": {
-      "Delay": 5,
-      "DefaultPoints": 250
+      "OnUseHandler": "Server.SkillHandlers.AnimalLore",
+      "Delay": "00:00:05",
+      "DefaultPoints": 250,
+      "StatAdvancements": {}
     },
     "AnimalTaming": {
-      "Delay": 10,
-      "IntAdvancement": {
-        "Chance": 80,
-        "PointsAmount": 4,
-        "PointsSides": 4,
-        "PointsBonus": 10
-      },
-      "DefaultPoints": 25
+      "OnUseHandler": "Server.SkillHandlers.AnimalTaming",
+      "Delay": "00:00:10",
+      "DefaultPoints": 25,
+      "StatAdvancements": {
+        "Int": {
+          "Chance": 0.8,
+          "MinGain": 8,
+          "MaxGain": 26
+        }
+      }
     },
     "Archery": {
-      "Delay": 0,
-      "StrAdvancement": {
-        "Chance": 50,
-        "PointsAmount": 4,
-        "PointsSides": 4,
-        "PointsBonus": 0
-      },
-      "DexAdvancement": {
-        "Chance": 60,
-        "PointsAmount": 4,
-        "PointsSides": 6,
-        "PointsBonus": 8
-      },
-      "DefaultPoints": 20
+      "Delay": "00:00:00",
+      "DefaultPoints": 20,
+      "StatAdvancements": {
+        "Str": {
+          "Chance": 0.5,
+          "MinGain": 8,
```

Comment:
Mmm 4 dice 4 sides should be Min 4 Max 16 no?

Reply:
Yeah true, good catch not sure what I was thinking there!

Reply:
Yeah not a big deal we will probably end up tweaking most of these values anyway after running the sims 😄